### PR TITLE
feat(tracking): add SBB tracking for topics and component events

### PR DESCRIPTION
**Pull request by danji90**
# How to

# Others

- [ ] It's not a hack or at least an unauthorized hack :).
- [ ] The images added are optimized.
- [ ] Everything in ticket description has been fixed.
- [ ] The author of the MR has made its own review before assigning the reviewer.
- [ ] The title means something for a human being and follows the [conventional commits](https://www.conventionalcommits.org/) specification.
- [ ] The title contains [WIP] if it's necessary.
- [ ] Labels applied. if it's a release? a hotfix?
- [ ] Tests added.

### CODE REVIEW

File: src/utils/useTrackTopic.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { useEffect } from "react";
```
Comment (friedjoff) [inline src/utils/useTrackTopic.js:1]: Diese Funktion würde ich auch in `trackingUtils.js` stecken, hängt ja stark zusammen.

File: src/utils/useTrackTopic.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+let prevUrl;
+
+function useTrackTopic(topicKey) {
+  const { i18n } = useTranslation();
+  useEffect(() => {
+    const env = getTrackingEnv();
+    if (env && topicKey) {
+      window.digitalDataLayer = window.digitalDataLayer || [];
+      window.digitalDataLayer.push({
+        pageInstanceID: env === "prod" ? "611081" : "611157",
```
Comment (friedjoff) [inline src/utils/useTrackTopic.js:14]: Damit klar ist, das hier noch was angepasst werden muss:
```suggestion
        pageInstanceID: env === "prod" ? "TODO: prod pageInstanceID" : "TODO: stag pageInstanceID",
```

**Review by friedjoff — CHANGES_REQUESTED** [review]
Sieht insgesamt gut aus, nur ein paar Kleinigkeiten ...

**Comment by danji90** [comment]
@friedjoff Please review again, especially the part about the zoom event tracking

**Pull request merged**